src/System.Resources.Reader/src/project.json

```diff
@@ -0,0 +1,17 @@
+{
+  "dependencies": {
+    "System.Collections": "4.0.10",
+    "System.Diagnostics.Contracts": "4.0.0",
+    "System.Diagnostics.Debug": "4.0.10",
+    "System.Diagnostics.Tools": "4.0.0",
+    "System.IO": "4.0.10",
+    "System.Resources.ResourceManager": "4.0.0",
+    "System.Runtime": "4.0.20",
+    "System.Runtime.Extensions": "4.0.10",
+    "System.Text.Encoding": "4.0.10",
+    "System.Threading": "4.0.10"
+  },
```

Comment:
How many of these dependencies need to be eliminated if you want to add this to the standard platform.